sample.yaml

```diff
@@ -372,37 +371,70 @@ scenes:
       environment:
         decision_environment:
           unstructured: >
-            EVAC has been delayed to 50 minutes out and can only take one patient.
+            The helicopter has been delayed and is still 50 minutes away.
+            You can also try to evac one casualty by ground immediately, but you're more likely to encounter hostiles.
           aid_delay:
-            - id: small_evac
+            - id: air_evac
               delay: 50
+              type: air
+              max_transport: 3
+            - id: ground_evac
+              delay: 1
+              type: ground
               max_transport: 1
     probe_config:
       - probe_id: adept-september-demo-probe-5
         description: Evacuation probe
     action_mapping:
-      - action_id: intend-evac-captain
+      - action_id: intend-evac-captain-air
         action_type: MOVE_TO_EVAC
-        intent_action: true # Specify the intent to evacuate
-        unstructured: Intend to move Katie to the road for transport
+        intent_action: true # Specify the intent to evacuate by air
+        unstructured: Intend to move Katie to the road for transport by helicopter
         character_id: Captain_01
-        parameters: {"evac_id": "small_evac"}
+        parameters: {"evac_id": "air_evac"}
         probe_id: adept-september-demo-probe-5
         choice: s1-p5-choice1
-      - action_id: intend-evac-marine
+      - action_id: intend-evac-marine-air
         action_type: MOVE_TO_EVAC
-        intent_action: true # Specify the intent to evacuate
-        unstructured: Intend to move Bob to the road for transport
+        intent_action: true # Specify the intent to evacuate by air
+        unstructured: Intend to move Bob to the road for transport by helicopter
         character_id: Marine_burns_01
-        parameters: {"evac_id": "small_evac"}
+        parameters: {"evac_id": "air_evac"}
         probe_id: adept-september-demo-probe-5
         choice: s1-p5-choice2
+      - action_id: intend-evac-captain-ground
+        action_type: MOVE_TO_EVAC
```

Comment:
If I change this action type to `MOVE_TO` it flags it as a warning but still says the file is valid. Should we be saying the file is valid if we know they have an issue like this?

Reply:
Nevermind, I see that it's just a warning because the character _may or may not_ be unseen

Reply:
Yeah there was a big problem with these because characters can switch to seen/unseen without the file explicitly defining it. So we're going to put this more on the TA1s to look through their files and make sure that the character is seen/unseen for the correct action types